lua/nvim-gps/init.lua

```diff
@@ -141,7 +171,7 @@ function M.get_location()
 
 	local filelang = ts_parsers.ft_to_lang(vim.bo.filetype)
 	local gps_query = ts_queries.get_query(filelang, "nvimGPS")
-	local transform = transform_lang[filelang]
+	local transform = transform_lang[filelang] or default_transform
```

Comment:
Nice trick! 😃 
I think we can remove these [lines](https://github.com/kassio/nvim-gps/blob/bf87a6eca3937326d6c29f5f1ccf18c7f18bd290/lua/nvim-gps/init.lua#L134)